benchmarks/benchmarks/sparse.py

```diff
@@ -207,15 +207,15 @@ def time_matvecs(self, format):
 
     def time_matvecs_inplace_1_0(self, fmt):
```

Comment:
Each of the inplace tests needs to be skipped if `fmt` is DIA or COO. It might be simpler to just make a second benchmark class, `MatvecsInplace`, which only tests CSR/CSC/BSR formats.